fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/DimensionFieldMapper.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.dimension;
+
+import java.util.Locale;
+
+/**
+ * Maps a Dimension Field value to a string.
+ */
+@FunctionalInterface
+public interface DimensionFieldMapper {
```

Comment:
I think `DimensionFieldNameMapper` is a pretty solid option, since it's clear that this maps a `DimensionField` into some representation of it's name.